src/catkin_pkg/package.py

```diff
@@ -357,18 +367,19 @@ def parse_package(path):
 
     with open(filename, 'r') as f:
         try:
-            return parse_package_string(f.read(), filename)
+            return parse_package_string(f.read(), filename, warnings)
```

Comment:
Please always pass kwargs with the explicit keyword instead of relying on the position. Same below.